src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/tests/PageList.test.js

```diff
@@ -90,8 +90,7 @@ beforeEach(() => {
 
 test('Render PageList', () => {
     const webspaceKey = observable.box('sulu');
-    // $FlowFixMe
-    const webspace = {};
+    const webspace = ({}: any);
```

Comment:
we cannot easily set a useful value here because the `redirectToWebspaceLocale()` method of the component will always trigger if the `webspace.localizations` are set

Reply:
also, it looks to me like the redirect logic of the component is not tested at the moment 🙈

Reply:
Can't we set some localizations and set the value of the `locale` observable in the component to one of the existing ones?

Reply:
unfortunately i dont think so. the redirect logic is executed in the constructor of the `PageList` component, therefore i cannot change any property of the component before it is executed: https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js#L103

furthermore, it looks like i am not able to set the `locale` via the router, as it is read from the router only after the redirect logic is executed. not sure if this is correct anyway, but i am also hesitant to change such things in this pr 🙈 

how should we proceed here?

Reply:
The router would have that already set, and this is considered in the `redirctToWebspace` function. So I would say the behavior is correct, although it might not be easy to test. So you would have to mock the `attributes` of the router I guess... But you are right, that's not part of this PR.